src/app/Scenes/Inbox/Components/Conversations/Messages.tests.tsx

```diff
@@ -146,10 +149,11 @@ describe("messages with order updates", () => {
     expect(extractText(tree.root)).toMatch(
       "To be covered by the Artsy Guarantee, always communicate and pay through the Artsy platform."
     )
+
     const toast = tree.root.findAllByType(Flex)[0]
     jest.advanceTimersByTime(150)
     expect(toast.props.opacity).toBe(1)
-    jest.advanceTimersByTime(5000)
+    jest.advanceTimersByTime(90000) // this number is weird, but i guess once Toast is moved to reanimated, this should be easier to use a much smaller number?
```

Comment:
```suggestion
    jest.advanceTimersByTime(900000) // this number is weird, but i guess once Toast is moved to reanimated, this should be easier to use a much smaller number?
```
lol this is stupid but I don't know whats up with the timers. the higher I go, the closer the animation goes to 0. so..

Reply:
I seriously do not know. I noticed the same exact thing. And the reverse too. At one point the assertion was trying to get to 1, and i kept bumping it up and up and up and then it got to `1.00000001` 😆 I'm like, ok, whatever, `xit`